includes/Modules/Ads.php

```diff
@@ -159,6 +162,24 @@ public function register_tag() {
 		$tag->register();
 	}
 
+	/**
+	 * Gets an array of debug field definitions.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
```

Comment:
Let's add a description for what is returned from this function to make it consistent with other method phpdocs.